keycastr/KCKeyboardTap.m

```diff
@@ -87,7 +87,20 @@ -(BOOL) installTapWithError:(NSError **)error {
     if (tapInstalled) {
         return YES;
     }
-    
+
+    // 10.9 and after
+    if (AXIsProcessTrustedWithOptions != NULL) {
+        NSDictionary *options = @{(__bridge id) kAXTrustedCheckOptionPrompt : @YES};
+        BOOL accessibilityEnabled = AXIsProcessTrustedWithOptions((__bridge CFDictionaryRef) options);
+
+        if (!accessibilityEnabled) {
+            if (error != NULL) {
+                *error = [self constructErrorWithDescription:@"Could not create keyDown event tap!"];
```

Comment:
We'll have to revisit the error messaging for this check failing. Also, this error interferes with the native dialog generated during the call to `AXIsProcessTrustedWithOptions()`